sdk/tests/resources/iterable-test.js

```diff
@@ -1,173 +0,0 @@
-/*
```

Comment:
Is this file recreated in the new directory? It's not showing up in this pull request.